stf/test/saros/stf/test/stf/editor/EditorByAliceBobTest.java

```diff
@@ -41,6 +45,11 @@ public void openAliceEditor() throws Exception {
     ALICE.remoteBot().editor(Constants.CLS1 + SUFFIX_JAVA).waitUntilIsActive();
   }
 
+  @After
+  public void tearDown() throws Exception {
+    leaveSessionPeersFirst(ALICE);
+  }
+
   @Test
   public void isEditorOpen() throws RemoteException {
```

Comment:
Looking at the whole tests the original author supposed they were executed in the order they appeared.

The name of the whole test class says nothing, looking at the test code it test the follow mode. So basically you could merge all tests into one and put it to the test class that is currently testing some follow mode stuff.